lms/static/sass/course/_info.scss

```diff
@@ -56,6 +56,95 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    div.upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      background-color: $white;
+      border: 1px solid $uxpl-gray-background;
+      @include border-left(0px);
+      width: 100%;
+      display: table;
+
+      .green-border {
+        width: 10px;
+        height: 100%;
+        margin: 0;
+        vertical-align: top;
+        background: $uxpl-green-base;
+        display: table-cell;
+      }
+
+      .upgrade-icon {
+        vertical-align: top;
+        margin: $baseline;
+        display: inline-block;
+        img {
+          max-height: 50px;
+        }
+      }
+
+      .upgrade-msg {
+        vertical-align: top;
+        margin: $baseline 0 $baseline 0;
+        display: inline-block;
+
+        h4 {
+          font-weight: font-weight(semi-bold);
+          font-size: font-size(large);
+        }
+        p {
+          margin-top: $baseline / 4;
+          font-weight: font-weight(normal);
+          font-size: font-size(base);
+        }
+
+        a:link, a:hover, a:visited, a:active {
```

Comment:
I don't like that this is necessary. I believe that the UX recommendation is that links are underlined by default now, so it would be good to see if this rule can be removed. This might only be possible on Pattern Library pages, though...